WeatherForecast/WeatherForecast/Model/LocationDelegate.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Protocol.swift
+//  WeatherForecast
+//
+//  Created by 김용재 on 2023/03/31.
+//
+
+import Foundation
+import CoreLocation
+
+protocol LocationDelegate: AnyObject {
+    func didUpdateLocation()
+}
```

Comment:
```suggestion
protocol CoreLocationManagerDelegate: AnyObject {
    func coreLocationManager(_ manager: CoreLocationManager, didUpdateLocations locations: [CLLocation])
}
```
보통 델리게이트 메소드의 경우는 보통 위처럼 델리게이트에 대한 source를 첫번째 파라미터로 만들고 기타 넘길 필요가 있는 값을 추가로 파라미터로 만듭니다. 참고하시고 왜 보통 그런지 이유도 생각해보시면 좋을 것 같아요!

Reply:
제이슨의 리뷰를 받고 Swift Style Guide에서 찾아본 delegate naming 관련 내용을 찾아 보았어요. 찾아본 결과 delegate methods에서는 delegate's source object를 첫번째 파라미터로 사용한다 하더라구요. 수정 완료했습니다!